sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector_node.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "pointcloud_preprocessor/distortion_corrector/distortion_corrector_node.hpp"
+
+#include "pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp"
+
+namespace pointcloud_preprocessor
+{
+/** @brief Constructor. */
+DistortionCorrectorComponent::DistortionCorrectorComponent(const rclcpp::NodeOptions & options)
+: Node("distortion_corrector_node", options)
+{
+  // initialize debug tool
+  {
```

Comment:
out of curiosity. why the scope?

Reply:
To be honest, I am also not sure why there are many baskets in the previous code. I removed some of them before. 
However, I think it is better to remove all of them in this case.
Thanks!

fixed in https://github.com/autowarefoundation/autoware.universe/pull/7137/commits/68d5cb9a1bbe87933110cac969b71536f4bffc6a